core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -668,6 +668,9 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     case "DATE_DIFF":
       unparseDateDiff(writer, call, leftPrec, rightPrec);
       break;
+    case "DATE_MOD":
+      unparseDateModule(writer, call, leftPrec, rightPrec);
```

Comment:
Can you please rename this to unparseDateModulo or unparseDateModulus

Reply:
Changes Done

Reply:
Looks ok